--- origsrc/transmission-1.93/qt/qtr.pro	2010-06-01 14:01:55.962085700 -0500
+++ src/transmission-1.93/qt/qtr.pro	2010-06-01 14:02:25.780791300 -0500
@@ -4,7 +4,7 @@ DESCRIPTION = "Transmission: a fast, eas
 VERSION = 1.60
 LICENSE = "GPL"
 
-target.path = /bin
+target.path = $$[QT_INSTALL_PREFIX]/bin
 INSTALLS += target
 
 CONFIG += qt thread debug link_pkgconfig
@@ -17,7 +17,9 @@ LIBS += $${TRANSMISSION_TOP}/libtransmis
 LIBS += $${TRANSMISSION_TOP}/third-party/dht/libdht.a
 LIBS += $${TRANSMISSION_TOP}/third-party/miniupnp/libminiupnp.a
 LIBS += $${TRANSMISSION_TOP}/third-party/libnatpmp/libnatpmp.a
-LIBS += $${TRANSMISSION_TOP}/third-party/libevent/.libs/libevent.a
+LIBS += -levent
+
+cygwin-g++ { LIBS += -lintl }
 
 TRANSLATIONS += transmission_en.ts transmission_ru.ts
 
