--- origsrc/transmission-2.42/qt/qtr.pro	2011-11-27 20:56:24.831986800 -0600
+++ src/transmission-2.42/qt/qtr.pro	2011-11-27 21:15:38.694984000 -0600
@@ -19,12 +19,11 @@ TRANSMISSION_TOP = ..
 INCLUDEPATH = $${EVENT_TOP}/include $${INCLUDEPATH}
 INCLUDEPATH += $${TRANSMISSION_TOP}
 LIBS += $${TRANSMISSION_TOP}/libtransmission/libtransmission.a
-exists( $${TRANSMISSION_TOP}/third-party/libutp/libutp.a ) { 
     LIBS += $${TRANSMISSION_TOP}/third-party/libutp/libutp.a
-}
 LIBS += $${TRANSMISSION_TOP}/third-party/dht/libdht.a
 LIBS += $${TRANSMISSION_TOP}/third-party/miniupnp/libminiupnp.a
 LIBS += $${TRANSMISSION_TOP}/third-party/libnatpmp/libnatpmp.a
 unix: LIBS += -L$${EVENT_TOP}/lib -lz -lrt
+cygwin-g++:LIBS += -lintl
 win32:DEFINES += QT_DBUS
 win32:LIBS += -levent-2.0 -lws2_32 -lintl
 win32:LIBS += -lidn -liconv -lwldap32 -liphlpapi
