--- origsrc/transmission-2.80/libtransmission/platform.c	2013-06-25 21:35:25.647458800 -0500
+++ src/transmission-2.80/libtransmission/platform.c	2013-07-07 15:13:28.167065800 -0500
@@ -16,7 +16,7 @@
   #include <ufs/ufs/quota.h> /* quotactl() */
  #elif defined (__sun)
   #include <sys/fs/ufs_quota.h> /* quotactl */
- #else
+ #elif !defined (__CYGWIN__)
   #include <sys/quota.h> /* quotactl() */
  #endif
  #ifdef HAVE_GETMNTENT
@@ -743,6 +743,7 @@ getblkdev (const char * path)
   return device;
 }
 
+#ifndef __CYGWIN__
 static int64_t
 getquota (const char * device)
 {
@@ -841,6 +842,7 @@ getxfsquota (char * device)
   return -1;
 }
 #endif /* HAVE_XQM */
+#endif /* CYGWIN */
 #endif /* WIN32 */
 
 static int64_t
@@ -848,7 +850,7 @@ tr_getQuotaFreeSpace (const struct tr_de
 {
   int64_t ret = -1;
 
-#ifndef WIN32
+#if !defined WIN32 && !defined __CYGWIN__
 
   if (info->fstype && !evutil_ascii_strcasecmp(info->fstype, "xfs"))
     {
