--- origsrc/transmission-2.83/libtransmission/platform-quota.c	2014-05-18 15:52:12.933912500 -0500
+++ src/transmission-2.83/libtransmission/platform-quota.c	2014-05-23 00:41:57.749791900 -0500
@@ -24,7 +24,7 @@
   #endif
  #elif defined (__sun)
   #include <sys/fs/ufs_quota.h> /* quotactl */
- #else
+ #elif !defined (__CYGWIN__)
   #include <sys/quota.h> /* quotactl() */
  #endif
  #ifdef HAVE_GETMNTENT
@@ -237,7 +237,7 @@ getquota (const char * device)
   freespace = limit - spaceused;
   return (freespace < 0) ? 0 : freespace;
 }
-#else
+#elif !defined(__CYGWIN__)
 static int64_t
 getquota (const char * device)
 {
@@ -344,7 +344,7 @@ tr_getQuotaFreeSpace (const struct tr_de
 {
   int64_t ret = -1;
 
-#ifndef WIN32
+#if !defined (WIN32) && !defined (__CYGWIN__)
 
   if (info->fstype && !evutil_ascii_strcasecmp(info->fstype, "xfs"))
     {
@@ -356,7 +356,7 @@ tr_getQuotaFreeSpace (const struct tr_de
     {
       ret = getquota (info->device);
     }
-#endif /* WIN32 */
+#endif /* WIN32 || Cygwin */
 
   return ret;
 }
